ios/FluentUI/Extensions/UITableViewCell+Extension.swift

```diff
@@ -5,7 +5,7 @@
 
 import UIKit
 
-extension UITableViewCell {
+public extension UITableViewCell {
```

Comment:
In what objects this extension will be used? Plain UITableViewCell instances?
 The functionality should be offered through the MSFTableViewCell (TableViewCell) internally.